### PR TITLE
Third milestone of your portfolio website : Create Contact Form

### PR DESCRIPTION
**Pull request by Roseokpe**
In this milestone, we added:

   - Contact form to the portfolio which include
       Text Area
       Input
       Button
 - Footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
...
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required placeholder="Text Input">
+        <input type="email" name="email" required id="email"  placeholder="test@gmail.com">
+        <textarea name="text" id="text" maxlength="300">Enter text here</textarea>
```
Comment (thecodechaser) [inline index.html]: - [ ] A user cannot submit the form if one or more of the fields is empty, Please also add a required field to the message so the user can not submit the form if the message field is empty.

File: index.html [hunk 2]
```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
...
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required placeholder="Text Input">
```
Comment (thecodechaser) [inline index.html]: - [ ] Please make sure the Name field accepts up to 30 chars, You can use the `maxLength` attribute.

File: index.html (hunk 1 above)
Comment (thecodechaser) [inline index.html]: - [ ] Please make sure the Message field accepts up to `500` chars, Please change the `maxLength` value to `500` from `300`.

File: index.html [hunk 3]
```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
...
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
```
Comment (thecodechaser) [inline index.html:199]: - [ ] Please add some paddings to the form fields and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design.

### Your design

image

### Figma design

image

File: index.html [hunk 4]
```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
...
+              <a href=""><img class="Twitter-icon" src="images/Icon-Twitter.png" alt=""></a>
+            </li>
+            <li>
+              <a href=""><img class="Medium-icon" src="images/Medium-icon.png" alt=""></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+        <p class="footer-text">test@gmail.com</p>
+    </footer>
```
Comment (thecodechaser) [inline index.html:232]: - [ ] Please use the correct color for `<hr>` and decrease the margin between social media icons and email addresses and make sure it matches with the Figma design.

### Your design

image

### Figma design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Roseokpe 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
- [ ] Please make sure that every member from your coding partner group has contributed to this project, I can only see your commits, I am requesting some changes Implement the changes with your group in pair programming we will check the commit history.
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
...
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
+        <input type="text" name="text" required maxlength="30" placeholder="Text Input">
+        <input type="email" name="email" required maxlength="30" id="email"  placeholder="test@gmail.com">
```
Comment (danmainah) [inline index.html]: - [ ] [OPTIONAL] Kindly remove the max length in your email field. It's not a requirement and it limits the number of characters in email, the best practice is to provide the format for the email input, which is already taken of by ` type="email"`.

**Review by danmainah — APPROVED** [review]
Hi @Roseokpe and @austync1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @Team


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**